package.json

```diff
@@ -30,7 +30,7 @@
     "@types/morgan": "^1.9.4",
     "big-integer": "^1.6.51",
     "body-parser": "^1.19.1",
-    "bootstrap": "^4.6.1",
+    "bootstrap": "^5.0.0",
```

Comment:
I don't think we can just update to bootstrap 5.0.0 like this without extensive testing!

Reply:
b222627401cd65ff742c9357c1d7f59aae349c6d